scripts/run-benchmarks.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -ex
```

Comment:
```suggestion
set -e
```

Reply:
The sensitive data is already obfuscated within the step `sendBenchmarks.prepareAndRun`, but I agree let's be less verbose to avoid any disclosure of data.